### ISSUE TITLE
When editing a spine scene

### ISSUE THREAD

**Issue by subsoap** [defold/editor2-issues#2721]
<hr/>

<table>
<tr><td>Build time</td><td>2019-08-27T11:22:32.157079</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>0e595467c47b581c444f54346d5d8642350a0040</td></tr>
<tr><td>Defold engine sha</td><td>1a8e53ae9c38a09b742d38dffc6a9f2efdbe6e97</td></tr>
<tr><td>Defold version</td><td>1.2.160</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/organizations/defold/issues/?query=id%3A"1e3f34d69d984f36850fd9f928f089e8"'>1e3f34d69d984f36850fd9f928f089e8</a></td></tr>
<tr><td>GPU</td><td>GeForce GTX 1060 6GB/PCIe/SSE2</td></tr>
<tr><td>GPU Driver</td><td>4.6.0 NVIDIA 431.60</td></tr>
<tr><td>Java version</td><td>11.0.1+13</td></tr>
<tr><td>OS arch</td><td>amd64</td></tr>
<tr><td>OS name</td><td>Windows 10</td></tr>
<tr><td>OS version</td><td>10.0</td></tr>
<table>

**Comment by subsoap** [comment]
```
clojure.lang.ExceptionInfo: handler ':cut' in context ':workbench' failed at ':enabled?' with message 'class clojure.lang.PersistentArrayMap cannot be cast to class java.util.Map$Entry (clojure.lang.PersistentArrayMap is in unnamed module of loader 'app'; java.util.Map$Entry is in module java.base of loader 'bootstrap')'
java.lang.ClassCastException: class clojure.lang.PersistentArrayMap cannot be cast to class java.util.Map$Entry (clojure.lang.PersistentArrayMap is in unnamed module of loader 'app'; java.util.Map$Entry is in module java.base of loader 'bootstrap')
```

```
clojure.lang.ExceptionInfo: handler ':set-camera-type' in context ':global' failed at ':state' with message 'Don't know how to create ISeq from: java.lang.Character'
java.lang.IllegalArgumentException: Don't know how to create ISeq from: java.lang.Character
```

**Comment by subsoap** [comment]
[owl.zip](https://github.com/defold/editor2-issues/files/3556889/owl.zip)

In this example, try setting the own spine model for the spine scene. The editor bugs out.

**Comment by matgis-king** [comment]
Duplicate of #2704.

Looks like they changed the exported file format significantly in Spine 3.8. We are working on a fix, but in the meantime we recommend you roll back Spine to 3.7 when exporting for Defold.

**Comment by AGulev** [comment]
Solved in 1.2.162
